docs/_data/prague-2022-config.yaml

```diff
@@ -28,10 +28,10 @@ buttons:
     #   link: /schedule
     # - text: Buy a Ticket!
     #   link: /tickets
-    - text: Sponsor us
-      link: /sponsors/prospectus
-    #   - text: Submit a Talk
-    #     link: /cfp
+    #- text: Sponsor us
+    #  link: /sponsors/prospectus
+    - text: Submit a Talk
```

Comment:
@plaindocs the CI is breaking, did we need to turn on the CFP flag or something?

Reply:
Oh maybe it's the URLs, let's see what happens :)

Reply:
But yeah, you'll need the CFP flag as well.

Reply:
@plaindocs I'd appreciate your help with anything flag/config related. I also think it's ok to replace the Welcome blog post button with the CFP, and then we'll have two buttons CFP + Sponsorship rather than the previous blog post... thoughts?